static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -0,0 +1,277 @@
+import {Fragment, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import HighlightTopRightPattern from 'sentry-images/pattern/highlight-top-right.svg';
+
+import Button from 'sentry/components/button';
+import DropdownMenuControl from 'sentry/components/dropdownMenuControl';
+import {MenuItemProps} from 'sentry/components/dropdownMenuItem';
+import IdBadge from 'sentry/components/idBadge';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import useOnboardingDocs from 'sentry/components/onboardingWizard/useOnboardingDocs';
+import useCurrentProjectState from 'sentry/components/replaysOnboarding/useCurrentProjectState';
+import {
+  generateDocKeys,
+  isPlatformSupported,
+} from 'sentry/components/replaysOnboarding/utils';
+import OnboardingStep from 'sentry/components/sidebar/onboardingStep';
+import SidebarPanel from 'sentry/components/sidebar/sidebarPanel';
+import {CommonSidebarProps, SidebarPanelKey} from 'sentry/components/sidebar/types';
+import {replayPlatforms} from 'sentry/data/platformCategories';
+import platforms from 'sentry/data/platforms';
+import {t, tct} from 'sentry/locale';
+import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
+import space from 'sentry/styles/space';
+import {Project} from 'sentry/types';
+import EventWaiter from 'sentry/utils/eventWaiter';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePrevious from 'sentry/utils/usePrevious';
+
+function ReplaysOnboardingSidebar(props: CommonSidebarProps) {
+  const {currentPanel, collapsed, hidePanel, orientation} = props;
+  const organization = useOrganization();
+
+  const isActive = currentPanel === SidebarPanelKey.ReplaysOnboarding;
+  const hasProjectAccess = organization.access.includes('project:read');
+
+  const {projects, currentProject, setCurrentProject} = useCurrentProjectState({
+    currentPanel,
+  });
+
+  if (!isActive || !hasProjectAccess || !currentProject) {
+    return null;
+  }
+
+  const items: MenuItemProps[] = projects
+    .reduce((acc: MenuItemProps[], project) => {
+      const itemProps: MenuItemProps = {
+        key: project.id,
+        label: (
+          <StyledIdBadge project={project} avatarSize={16} hideOverflow disableLink />
+        ),
+        onAction: function switchProject() {
+          setCurrentProject(project);
+        },
+      };
+
+      if (currentProject.id === project.id) {
```

Comment:
Is this so selected project is at the top? If so, does the `sort()` invalidate this?